### PR TITLE
Redesign map

### LINKED ISSUE #207: /map initial zoom
We should fix the map initial zoom because it seems to me a bit awkward to have blue fields around the map. This is my idea and I would like to have a bigger initial zoom like here: http://gameofthrones.com/game-of-thrones/season-5/episode-10/map/location/5/braavos-free-city

@mbandle @AlexBeischl 
How can we fix this on our side? Thanks!

### PR DESCRIPTION
**Pull request by jorjo1**
# Dude, pimp my map

Closes #207 #277 
before:
image
after:
image

### CODE REVIEW

**Comment by mammuth** [comment]
Waiting until @jorjo1 fixes an introduced bug in the detail page.

**Comment by jorjo1** [comment]
I stumbled upon some problems with the elements beneath the map, because I set the height to the wrong map-wrapper. 
It's now ready to go! :tada: 
image

**Comment by yashha** [comment]
Looks really awesome
But now the map on characterspage and startpage is broken :(
And the area where the loaded characters are displayed isnt scrollable anymore

**Comment by yashha** [comment]
It's still buggy so first fix it! :P
Or shell I fix it? ^^

**Comment by jorjo1** [comment]
Just tell me what to fix

**Comment by yashha** [comment]
As I wrote before ;)

> But now the map on characterspage and startpage is broken :(
> And the area where the loaded characters are displayed isnt scrollable anymore

**Comment by yashha** [comment]
Ok map on characterspage and startpage is **not** broken
I don't know when I tested it, it was, sry

**Comment by jorjo1** [comment]
It is probably because of an older version :8ball: 

edit: oh yeah, the characters in the sidebar are not scrolled when too many! Nice catch. Let me fix it

File: app/components/common/MapComp/MapComp.jsx [hunk 1]
```diff
@@ -113,10 +117,12 @@ export default class MapComp extends Component {
     if (patt.test(end)) {
       end = (patt.exec(end)[1]) *  patt.exec(end)[2];
     } else { end = 50; }
-    return [parseInt(begin),parseInt(end)];
+   return [parseInt(begin),parseInt(end)];
   }
 
+    
   render() {
+    console.log(this.props.pagex);/*eslint no-console:0,no-undef:0*/
```
Comment (yashha) [inline app/components/common/MapComp/MapComp.jsx]: Please remove ;)

**Comment by yashha** [comment]
Passt!
Eine sache wäre noch die scrollbar zu stylen, aber sonst ist das top.
Ich merge mal.

**Comment by jorjo1** [comment]
Toitoitoi! 
Yes, just merge it. I know about this scrollbar, but I'm tired right now and we have to close this PR sometime :dancer:

**Comment by sacdallago** [comment]
I love the first screen :D

**Pull request merged**